no_log.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- hosts: all
+  gather_facts: false
+  tasks:
+    - name: task with stdout
+      shell: echo {{ secret_value }}
```

Comment:
Passed in as a launch-time variable. Looked at your other playbooks in `jlaska/ansible-playbooks` and that seemed to be the established way of doing things.